dockers/docker-lldp-sv2/lldpmgrd

```diff
@@ -135,34 +134,21 @@ class LldpManager(object):
             if not port_alias:
                 log_info("Unable to retrieve port alias for port '{}'. Using port name instead.".format(port_name))
                 port_alias = port_name
+            
+            # Get the port description. If None or empty string, we'll skip this configuration 
+            port_desc = port_table_dict.get("description")
+
         else:
             log_error("Port '{}' not found in {} table in Config DB. Using port name instead of port alias.".format(port_name, swsscommon.CFG_PORT_TABLE_NAME))
             port_alias = port_name
 
         lldpcli_cmd = "lldpcli configure ports {0} lldp portidsubtype local {1}".format(port_name, port_alias)
 
-        # Retrieve all entires for this port from the Device Neighbor table
-        device_neighbor_table = swsscommon.Table(self.config_db, swsscommon.CFG_DEVICE_NEIGHBOR_TABLE_NAME)
-        (status, fvp) = device_neighbor_table.get(port_name)
-        if status:
-            # Convert list of tuples to a dictionary
-            device_neighbor_table_dict = dict(fvp)
-
-            # Get neighbor host name and port name
-            neighbor_hostname = device_neighbor_table_dict.get("name")
-            neighbor_portname = device_neighbor_table_dict.get("port")
-
-            # If we sucessfully obtained the neighbor's host name and port name, append a port description to the command
-            if neighbor_hostname and neighbor_portname:
-                lldpcli_cmd += " description {0}:{1}".format(neighbor_hostname, neighbor_portname)
-            else:
-                if not neighbor_hostname:
-                    log_info("Failed to retrieve neighbor host name for port '{}'. Not adding port description.".format(port_name))
-
-                if not neighbor_portname:
-                    log_info("Failed to retrieve neighbor port name for port '{}'. Not adding port description.".format(port_name))
+        # if there is a description available, also configure that
+        if port_desc:
+            lldpcli_cmd += " description {}".format(port_desc)
         else:
-            log_info("Unable to retrieve neighbor information for port '{}'. Not adding port description.".format(port_name))
+            log_info("Unable to retrieve description for port '{}'. Not adding port description")
```

Comment:
Why remove the string formatting argument? Now there is a formating specifier without a matching argument.

Reply:
Hi @jleveque , can you approve?